FilePersistence/src/version_control/GitRepository.cpp

```diff
@@ -1074,8 +1074,7 @@ git_commit* GitRepository::parseCommit(QString revision) const
 
 QString GitRepository::oidToQString(const git_oid* oid) const
 {
-	char sha1[41];
-	sha1[40] = '\0';
+	char sha1[GIT_OID_HEXSZ + 1] = {0};
```

Comment:
Well, since you're doing this, using default initialization `{}` is even nicer, I think.